### PR TITLE
Feature/remove reagent atom from theme

### PR DESCRIPTION
**Pull request by flexsurfer**
remove reagent atom from theme

based on approved https://github.com/status-im/status-mobile/pull/19660

so only one commit should be reviewed  https://github.com/status-im/status-mobile/pull/19706/commits/1f7018b6cc0939f0532d84e50cecf8864ba90fd1

for QA: theme of all components affected

### CODE REVIEW

File: scripts/lint/re-frame-in-quo-components.sh [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env sh
 
-INVALID_CHANGES=$(grep -E -r '(re-frame/dispatch|rf/dispatch|re-frame/subscribe|rf/subscribe|rf/sub|<sub|>evt|status-im\.)' --include '*.cljs' --include '*.clj' './src/quo')
+INVALID_CHANGES=$(grep -E -r '(/atom|re-frame/dispatch|rf/dispatch|re-frame/subscribe|rf/subscribe|rf/sub|<sub|>evt|status-im\.)' --include '*.cljs' --include '*.clj' './src/quo')
 
 if test -n "$INVALID_CHANGES"; then
-    echo "WARNING: re-frame, status-im are not allowed in quo components"
+    echo "WARNING: re-frame, status-im, regent atoms are not allowed in quo components"
```
Comment (ibrkhalil) [inline scripts/lint/re-frame-in-quo-components.sh]: `regent -> reagent`

**Review by ibrkhalil — APPROVED** [review]
🚀

**Review by clauxx — APPROVED** [review]
LGTM

only checked the theme & navigation setup tho 😟

**Comment by mariia-skrypnyk** [comment]
Hi @flexsurfer !

Thanks for tons of work!

A few minor point described below 👇

**ISSUE 1: status bar is black in a dark theme**

**Devices:** iOS 13 (17.3.1) Samsung A34 (14.0)

**Steps:**

1. Login to app
2. Go to Profile-> Appearance
3. Change theme to dark
4. Return to Messages or Community screens

**Actual result:**
Screenshot 2024-04-18 at 16 05 54

**Expected result:** https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?type=design&node-id=0-1&mode=design&t=ZZJtrLI3PA3FmD16-0

**Comment by mariia-skrypnyk** [comment]
**ISSUE 2: black stripe on Profile Settings screens**

Devices: iOS 13 (17.3.1) Samsung A34 (14.0)

**Steps:**

1. Login to app
2. Go to Profile-> Appearance
3. Change theme to dark
4. Open such screens: Advanced, Notifications, Appearence, About.

**Actual result:**

Screenshot 2024-04-18 at 16 11 49

**Comment by flexsurfer** [comment]
thanks @mariia-skrypnyk all will be fixed in the followup PRs

**Pull request merged**